data/human/news.txt

```diff
@@ -423,6 +432,7 @@ news "human resources worker"
 			"We always put our employees first. But sometimes they do things that are bad for their careers. Like, accusing a high-ranking manager of harassment."
 			"I once had to find a delicate way to explain to a worker that he smelled bad. Like, so bad that no one could bear to be in a small meeting room with him."
 			"For most jobs, it's essential to know how to get along with others and be a team player. But I bet a freelance pilot like you never needs to worry about that."
+			"Most people overestimate or underestimate their own capabilities. The people who overestimate their capabilities usually don't last long in their jobs."
```

Comment:
What about people who underestimate their own capabilities?

Reply:
I'll split this one into two versions.